res/css/views/right_panel/_ThreadPanel.scss

```diff
@@ -125,18 +126,6 @@ limitations under the License.
         }
     }
 
-    // chat panel with a maximized widget
-    &.mx_TimelineCard {
-        .mx_GroupLayout {
-            .mx_EventTile {
-                .mx_MessageTimestamp {
-                    position: absolute;
-                    width: $MessageTimestamp_width;
```

Comment:
These are partial revert of https://github.com/matrix-org/matrix-react-sdk/pull/8337/commits/addd6a723f2a49192c26dc4e2397de9157962646. They are not required thanks to the original properties on [_GroupLayout.scss](https://github.com/matrix-org/matrix-react-sdk/blob/472222c195d5d953d673519f7ba8e00cf56fc4ca/res/css/views/rooms/_GroupLayout.scss#L32).